src/AdamWathan/Form/Elements/Select.php

```diff
@@ -5,10 +5,13 @@ class Select extends FormControl
 
     private $options;
     private $selected;
+    private $name;
 
     public function __construct($name, $options = array())
     {
+        $this->name = $name;
```

Comment:
Instead of storing this again, just access it with `$this->attributes['name']`. Might be nice in the future to access attributes through `__get` but could be a separate PR.